SPECS/xen.spec

```diff
@@ -1305,6 +1330,15 @@ touch %{_rundir}/reboot-required.d/%{name}/%{version}-%{hv_rel}
 %{?_cov_results_package}
 
 %changelog
+* Thu Apr 11 2024 David Morel <david.morel@vates.tech> - 4.13.5-9.40.1
+- Security update, synced from hotfix XS82ECU1062
+- *** Upstream changelog ***
+- * Mon Apr  8 2024 Andrew Cooper <andrew.cooper3@citrix.com> - 4.13.5-9.40
+- - Fixes for:
+-   - XSA-454 CVE-2023-46842
+-   - XSA-455 CVE-2024-31142
+-   - XSA-456 CVE-2024-2201
+
 * Wed Mar 13 2024 David Morel <david.morel@vates.tech> - 4.13.5-9.39.1
 - Security update, synced from hotfix XS82ECU1061
 - Replace our fix for XSA-449
```

Comment:
The lines below were fixed (swapped 2 XSA numbers) in the original SRPM, then lost in the merge commit.

Reply:
Updated